index.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
         alphabetize: {
           order: "asc"
         },
+        pathGroupsExcludedImportTypes: ["builtin"],
```

Comment:
I've read the docs on this but i'm still not entirely sure what omitting `external` will actually result in. 🤔

Reply:
I'm not totally clear, but without this line the imports are ordered incorrectly.

Reply:
Do we lose the ordering for external/npm dependencies?

Reply:
Double checked – ordering of external dependencies continues to work correctly!